Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardTableViewController.swift

```diff
@@ -183,14 +178,24 @@ extension CreditCardTableViewController: UITableViewDelegate,
             withIdentifier: HostingTableViewCell<CreditCardItemRow>.cellIdentifier) as? HostingTableViewCell<CreditCardItemRow> else {
             return UITableViewCell()
         }
-
+        let creditCardCount = viewModel.creditCards.count
         let creditCard = viewModel.creditCards[indexPath.row]
         let creditCardRow = CreditCardItemRow(
             item: creditCard,
-            isAccessibilityCategory: UIApplication.shared.preferredContentSizeCategory.isAccessibilityCategory)
+            isAccessibilityCategory: UIApplication.shared.preferredContentSizeCategory.isAccessibilityCategory,
+            shouldShowSeparator: indexPath.row < creditCardCount - 1 && creditCardCount > 1,
```

Comment:
Addressing case for when there is only one card or we are on the last card so we should not show the separator